DataConnectors/AWS-S3/Enviornment/EnviornmentConstants.ps1

```diff
@@ -0,0 +1,2 @@
+$script:SentinelClientId = '21f935c0-8092-4b62-a772-5a2afd714569'
+$script:regionForSecurity = 'Fairfax'
```

Comment:
maybe change name to cloud env and change to customer facing names , public and gov